src/plugins/intel_gpu/src/graph/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -32,12 +32,33 @@ void experimental_detectron_roi_feature_extractor_inst::copy_rois_input_to_secon
 
 template<typename ShapeType>
 std::vector<layout> experimental_detectron_roi_feature_extractor_inst::calc_output_layouts(
-        experimental_detectron_roi_feature_extractor_node const& /*node*/, const kernel_impl_params& impl_param) {
+        experimental_detectron_roi_feature_extractor_node const& node, const kernel_impl_params& impl_param) {
     layout rois_layout = impl_param.get_input_layout(0);
     layout data_layout = impl_param.get_input_layout(1);
+    auto desc = impl_param.typed_desc<experimental_detectron_roi_feature_extractor>();
```

Comment:
That change looks overcomplicated. As I understand, the original issue is related to batch() and feature() methods usage for dynamic shapes case. So I suggest the following replacement:
```cpp
auto num_rois = rois_layout.get_partial_shape()[0];
auto num_channels = data_layout.get_partial_shape()[1];
```